addons/im_livechat/static/src/js/im_livechat_channel_form_controller.js

```diff
@@ -0,0 +1,45 @@
+odoo.define('im_livechat.ImLivechatChannelFormController', function (require) {
+'use strict';
+
+const FormController = require('web.FormController');
+
+const ImLivechatChannelFormController = FormController.extend({
+    events: Object.assign({}, FormController.prototype.events, {
+        'click .o_im_livechat_channel_form_button_colors_reset_button': '_onClickLivechatButtonColorsResetButton',
+        'click .o_im_livechat_channel_form_chat_window_colors_reset_button': '_onClickLivechatChatWindowColorsResetButton',
+    }),
+    /**
+     * @private
+     */
+    async _onClickLivechatButtonColorsResetButton() {
+        this.$('[name="button_background_color"] .o_field_color').css('background-color', "#878787");
+        this.$('[name="button_text_color"] .o_field_color').css('background-color', "#FFFFFF");
+        const result = await this.model.notifyChanges(
+            this.handle,
+            {
+                button_background_color: "#878787",
+                button_text_color: "#FFFFFF"
+            }
+        );
+        this.renderer.updateState(this.model.get(this.handle), { fieldNames: result });
```

Comment:
You could use a `const` to keep default values in order to avoid the rewriting.
Like : 
```javascript
const default_background_color = "#878787";
```
and then use it in `.css` and `notifyChanges` (the same for other colors)

Reply:
You could even do a function like below, called by these two but maybe it's overkill.

```javascript
async _updateColors(colorValues){
    for(let name in colorValues){
        this.$(`[name="${name}"] .o_field_color`).css('background-color', colorValues[name]);
    }
    const result = await this.model.notifyChanges(this.handle, colorValues);
    this.renderer.updateState(this.model.get(this.handle), { fieldNames: result });
}

async onClickLivechatButtonColorsResetButton() {
    this._updateColor({
        button_background_color: "#878787",
        button_text_color: "#FFFFFF"
    });
}
```

And something similar for `onClickLivechatChatWindowColorsResetButton`